SU-SWS/decanter
# Updated packages and configuration files.

# READY FOR REVIEW

# Summary
- Updates package dependencies, dot files, and other operation related configuration.

# Needed By (Date)
- All of this is in various parts of other PRs so having it all in one place is nice but not urgent. 

# Urgency
- Low

# Steps to Test

1. Check out this branch and delete your node_modules directory.
2. Run `nvm install 11.14.0`
3. Run `nvm use`
4. Run `npm install`
5. Run `npm run build`
6. Run `npm run styleguide`
7. For bonus points run a few more commands.

# Affected Projects or Products
- Just operational things.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
